style.css

```diff
@@ -106,3 +106,220 @@ button {
     display: none;
   }
 }
+
+/* class box-job, box-img, box-title, box-tags, btn-showup  */
+
+/* Styling My recent work section */
+.jobs {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 200px repeat(auto, 6fr);
+  text-align: center;
+  width: 90%;
+  margin: 0 auto;
+}
+
+.jobs h1:first-child {
+  padding: 1em;
+}
+
+.jobs h1:first-child::after {
+  content: "";
+  height: 4px;
+  width: 50px;
+  margin: 0 auto;
+  display: block;
+  position: relative;
+  top: 20px;
+  z-index: -1;
+  background-color: var(--color-primary);
+  border-radius: 9px;
+}
+
+.box-job {
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  padding: 0;
+  margin: 1em auto;
+  width: 330px;
+  height: 480px;
+  border: 1px #d0d9d4;
+  border-radius: 8 px;
+  transition: box-shadow 0.3s ease-in-out;
+}
+
+.box-job:hover {
+  box-shadow: var(--shadow);
+}
+
+.box-img {
+  overflow: hidden;
+  box-sizing: border-box;
+}
+
+.box-img img {
+  width: 100%;
+  min-height: 250px;
+  object-fit: cover;
+}
+
+.box-tags,
+.skill-tags {
+  list-style: none;
+  padding: 0%;
+  display: flex;
+  justify-content: center;
+}
+
+.box-job > .box-tags > li {
+  background: #ebf0ee;
+  font-size: 12px;
+  padding: 0.5em;
+  margin: 0 0.5em;
+  border-radius: 4px;
+}
```

Comment:
- [ ] For these tech tags in the project cards, please fix the color which should be `#3A4A42` , also fix the color of the project card's title i.e. 'Multi-Post_stories' 
![image](https://user-images.githubusercontent.com/97521557/234957797-8967e59e-704c-49c1-9803-7668c12660b7.png)
 figma :point_down: 
![image](https://user-images.githubusercontent.com/97521557/234957932-5b99d960-0bf5-4ebe-be6d-48bd13052b02.png)